CoffeeChat/Controller/EditLocationAvailabilityViewController.swift

```diff
@@ -0,0 +1,296 @@
+//
+//  EditLocationAvailabilityViewController.swift
+//  CoffeeChat
+//
+//  Created by Amy Chin Siu Huang on 11/19/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+// MARK: - Location UICollectionView Header
+private class LocationHeaderLabel: UICollectionReusableView {
+
+    private let label = UILabel()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        label.font = ._16CircularStdBook
+        addSubview(label)
+        label.snp.makeConstraints { make in
+            make.center.equalToSuperview()
+        }
+    }
+
+    required init?(coder: NSCoder) { fatalError("init(coder:) has not been implemented") }
+
+    func configure(with text: String) { label.text = text }
+
+}
+
+class EditLocationAvailabilityViewController: UIViewController {
+
+    // MARK: - Private View Vars
+    private let backButton = UIButton()
+    private var locationsCollectionView: UICollectionView!
+    private let saveBarButtonItem = UIBarButtonItem()
+    private let scheduleLocationLabel = UILabel()
+    private let scheduleLocationSubLabel = UILabel()
+
+    // MARK: - Location CollectionView Sections
+    private enum LocationSection {
+        case campus([String])
+        case ctown([String])
+    }
+
+    // MARK: - Location Data Vars
+    private let campusReuseId = "campusReuseIdentifier"
```

Comment:
For reuse identifiers, our convention now is that we declare them as static variables inside the collection view cell that we will be using! So for example, inside of `SchedulingPlaceCollectionViewCell` file, we say `static let campusReuseId = "campusReuseIdentifier"`. And then in this file, we refer to that variable as `SchedulingPlaceCollectionViewCell.campusReuseId`